versions.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "3.72"
+      version = "5.3.0"
```

Comment:
Latest version is 5.4.0: https://registry.terraform.io/providers/hashicorp/aws/latest/docs